src/unstract/sdk/index.py

```diff
@@ -38,11 +39,6 @@ def get_text_from_index(
         embedding_li = embedd_helper.get_embedding(
             adapter_instance_id=embedding_type
         )
-        if embedding_li is None:
-            self.tool.stream_log(
-                f"Error loading {embedding_type}", level=LogLevel.ERROR
-            )
-            raise SdkError(f"Error loading {embedding_type}")
```

Comment:
I can see that in few different places this sort of existing error throwing parts are removed. Can you elaborate why it is safe to do so? May be you can add same points in the PR description itself.

Reply:
In a recent PR we added the error throwing logic within the `get_llm_instance()` or `get_embedding_instance()` methods itself instead of returning None - this would help avoid repeated error handling in all places of use